ui-tests/src/test/java/io/syndesis/qe/steps/connections/wizard/ConnectionWizardSteps.java

```diff
@@ -0,0 +1,41 @@
+package io.syndesis.qe.steps.connections.wizard;
+
+import static org.junit.Assert.assertThat;
+
+import static org.hamcrest.Matchers.is;
+
+import static com.codeborne.selenide.Selenide.$;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import cucumber.api.java.en.And;
+import io.syndesis.qe.instances.SharedInstances;
+import io.syndesis.qe.pages.connections.wizard.ConnectionWizard;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ConnectionWizardSteps {
+
+    @Autowired
+    private ConnectionWizard wizard;
+
+    @And("^(\\w+) navigates to the next Connection wizard step \"([^\"]*)\"$")
+    public void navigateToNextWizardStep(String user, String step) {
+        switch(step) {
+            case "Config":
```

Comment:
Isn't there missing something? What's the purpose of this case?

Reply:
I'm not sure, I have deleted the switch structure. I'll get back to it later. New task created for this one. https://github.com/syndesisio/syndesis-qe/issues/372